runtime/tools/python/ttrt/binary/module.cpp

```diff
@@ -27,7 +31,23 @@ PYBIND11_MODULE(_C, m) {
       .def_property_readonly("file_identifier",
                              &tt::runtime::Binary::getFileIdentifier)
       .def("as_json", &tt::runtime::Binary::asJson)
-      .def("store", &tt::runtime::Binary::store);
+      .def("store", &tt::runtime::Binary::store)
+      .def("get_debug_info_golden", [](tt::runtime::Binary &binary,
+                                       std::string &loc) {
+        const ::tt::target::GoldenTensor *goldenTensor =
+            binary.getDebugInfoGolden(loc);
+        if (goldenTensor == nullptr) {
+          return std::vector<float>();
+        }
+
+        int totalDataSize = std::accumulate((*goldenTensor->shape()).begin(),
+                                            (*goldenTensor->shape()).end(), 1,
+                                            std::multiplies<int64_t>());
+        std::vector<float> dataVec(totalDataSize);
+        std::memcpy(dataVec.data(), goldenTensor->data()->data(),
+                    totalDataSize * sizeof(float));
+        return dataVec;
```

Comment:
I think this workaround is OK for now, we can come up with something more robust when the time comes.

Reply:
sounds good